src/botPage/view/blockly/index.js

```diff
@@ -60,9 +60,7 @@ const disposeBlocksWithLoaders = () => {
 
 const marketsWereRemoved = xml => {
     if (!Array.from(xml.children).every(block => !removeUnavailableMarkets(block))) {
-        if (isProduction()) {
-            TrackJS.track('Invalid financial market');
-        }
+        trackJSTrack({ code:'Invalid financial market' });
```

Comment:
It's better to pass `message` and `code` to the `trackJSTrack` method.  
```suggestion
        const error = new TrackJSError(....)
        trackJSTrack(error);
```

Reply:
@mehrzad-fs I totally agree with you, but probably here the idea was to pass in all cases only **code:'Invalid financial market'** for any reason...